app/src/main/java/org/oppia/android/app/help/thirdparty/LicenseTextViewerFragmentPresenter.kt

```diff
@@ -0,0 +1,62 @@
+package org.oppia.android.app.help.thirdparty
+
+import android.text.Html
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.appcompat.app.AppCompatActivity
+import androidx.fragment.app.Fragment
+import org.oppia.android.app.fragment.FragmentScope
+import org.oppia.android.databinding.LicenseTextViewerFragmentBinding
+import javax.inject.Inject
+import org.oppia.android.R
+
+/** The presenter for [LicenseListFragment]. */
+@FragmentScope
+class LicenseTextViewerFragmentPresenter @Inject constructor(
+  private val activity: AppCompatActivity,
+  private val fragment: Fragment,
+) {
+  private lateinit var binding: LicenseTextViewerFragmentBinding
+
+  /** Handles onCreateView() method of the [LicenseTextViewerFragment]. */
+  fun handleCreateView(
+    inflater: LayoutInflater,
+    container: ViewGroup?,
+    dependencyIndex: Int,
+    licenseIndex: Int
+  ): View? {
+    val viewModel = getLicenseTextViewModel(activity, dependencyIndex, licenseIndex)
+
+    binding = LicenseTextViewerFragmentBinding.inflate(
+      inflater,
+      container,
+      /* attachToRoot= */ false
+    )
+
+    binding.let {
+      it.lifecycleOwner = fragment
+      it.viewModel = viewModel
+    }
+    val dependenciesWithLicenseTexts = activity.resources.obtainTypedArray(
+      R.array.third_party_dependency_license_texts_array
+    )
+    val licenseTextsArrayId = dependenciesWithLicenseTexts.getResourceId(
+      dependencyIndex,
+      0
+    )
+    val licenseTextsArray: Array<String> = activity.resources.getStringArray(
+      licenseTextsArrayId
+    )
+    binding.licenseTextTextview.text = Html.fromHtml(licenseTextsArray[licenseIndex])
```

Comment:
I even tried to set the license text directly from the fragment, but it still did not display the license text in the expected format.
<img src="https://user-images.githubusercontent.com/54636525/127386497-c8645f50-9583-4802-9a0e-d81075f6b208.jpg" width=300>

Reply:
Issue resolved.